modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/graphql/v1_0/test/ObjectDefinitionGraphQLTest.java

```diff
@@ -206,6 +208,101 @@ public void tearDown() throws PortalException {
 		ObjectEntryLocalServiceUtil.deleteObjectEntry(_parentObjectEntry);
 	}
 
+	@Test
+	@TestInfo("LPD-49283")
+	public void testAddAndGetObjectEntryWithMultiselectPicklistField()
+		throws Exception {
+
+		ListTypeDefinition listTypeDefinition =
+			ListTypeDefinitionLocalServiceUtil.addListTypeDefinition(
+				null, TestPropsValues.getUserId(),
```

Comment:
I added a commit to fix the implementation, as we would cover any array of elements instead of just an array of items that are not primive. Please, have a look at the commit and let me know whatever you consider